analyzers/src/SonarAnalyzer.Common/Helpers/SonarAnalysisContext.cs

```diff
@@ -182,8 +189,7 @@ private static bool IsTestProject(TryGetValueDelegate<ProjectConfigReader> tryGe
                 : projectType == ProjectType.Test;  // Scanner >= 5.1 does authoritative decision that we follow
         }
 
-        private static SourceTextValueProvider<bool> CreateAnalyzeGeneratedProvider(string language) =>
-            new SourceTextValueProvider<bool>(x => PropertiesHelper.ReadAnalyzeGeneratedCodeProperty(ParseXmlSettings(x), language));
+        private static SourceTextValueProvider<bool> CreateAnalyzeGeneratedProvider(string language) => new(x => PropertiesHelper.ReadAnalyzeGeneratedCodeProperty(ParseXmlSettings(x), language));
```

Comment:
Not trivial, needs a new line
```suggestion
        private static SourceTextValueProvider<bool> CreateAnalyzeGeneratedProvider(string language) => 
            new(x => PropertiesHelper.ReadAnalyzeGeneratedCodeProperty(ParseXmlSettings(x), language));
```